src/client/containers/AddPeopleToRoom/AddPeopleToRoom.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import AddPeopleToRoomForm from '../../components/AddPeopleToRoom/AddPeopleToRoomForm';
+import '../../components/AddPeopleToRoom/AddPeopleToRoom.css';
+import AddNewRoom from '../AddNewRoom/AddNewRoom';
+
+export default function AddPeopleToRoom() {
+  const [users, setUsers] = useState([]); // this is use to control the data from api
+  const [addUsers, setAddUsers] = useState([]); //  this is use to add user to usersgroup
+  const [input, setInput] = useState([]); //  this is use to handle the input from user to search out
+  const [toAddRoom, setToAddRoom] = useState('false'); //  thi is use to move to next page
```

Comment:
Do not use `'true'` and `'false'` which are strings, use `true` and `false` which are booleans.